### PR TITLE
added video upload feature

### PR DESCRIPTION
**Pull request by JantuDeb**
## Upload video
- can upload a video with title, description, thumbnail, video, etc.
## Fix
- fixed some typos in the function name
- removed the search bar from all the pages except the home page.
- fixed the login persistence issue

image

### CODE REVIEW

File: src/components/navbar/Navbar.css [hunk 1]
```diff
@@ -52,6 +52,10 @@ header {
 .btn-search-mobile{
     display: none;
 }
+
+.upload-video.active{
+  color: red;
```
Comment (t007rushi) [inline src/components/navbar/Navbar.css:57]: ```color variable```

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -7,32 +7,24 @@ import { FiSearch } from "react-icons/fi";
...
 import { useAuth } from "../../context/auth/AuthContext";
 
-const Navbar = ({ hideHamburgerMenu , toogle}) => {
+const Navbar = ({ hideHamburgerMenu, toogle }) => {
   const { authState } = useAuth();
-
-  const handleClick = () => toogle((v) => !v);
-  const navigate = useNavigate();
   const { pathname } = useLocation();
+  const handleClick = () => toogle((v) => !v);
```
Comment (t007rushi) [inline src/components/navbar/Navbar.jsx:20]: have clear naming v? video would be understandable

**Review by sameep-s — APPROVED** [review]
Code is clean 
folder structure is good
you can merge.

**Review by t007rushi — APPROVED** [review]
UPLOAD VIDEO🤯
warsi-circuit-bhai-yeh-toh-god-hai-yaarGIF

File: src/components/navbar/Navbar.css (hunk 1 above)
Comment (JantuDeb) [reply src/components/navbar/Navbar.css:57]: Bhul gaya fir se

File: src/components/navbar/Navbar.jsx (hunk 2 above)
Comment (JantuDeb) [reply src/components/navbar/Navbar.jsx:20]: It is a Boolean value so, I will change it to `isTrue`

**Pull request merged**